docs-en/chapter_preface/suggestions.md

```diff
@@ -2,186 +2,186 @@
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
     /**
-     * multi-line
-     * marginal notes
+     * Multiline
+     * comments
      */
     ```
 
 === "TS"
 
     ```typescript title=""
-    /* Header comments for labeling functions, classes, test samples, etc. */
+    /* Header comments for labeling functions, classes, test samples, etc */
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
     /**
-     * multi-line
-     * marginal notes
+     * Multiline
+     * comments
      */
     ```
 
 === "Dart"
 
     ```dart title=""
-    /* Header comments for labeling functions, classes, test samples, etc. */
+    /* Header comments for labeling functions, classes, test samples, etc */
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
     /**
-     * multi-line
-     * marginal notes
+     * Multiline
+     * comments
      */
     ```
 
 === "Rust"
 
     ```rust title=""
-    /* Header comments for labeling functions, classes, test samples, etc. */
+    /* Header comments for labeling functions, classes, test samples, etc */
 
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
     /**
-     * multi-line
-     * marginal notes
+     * Multiline
+     * comments
      */
     ```
 
 === "C"
 
     ```c title=""
-    /* Header comments for labeling functions, classes, test samples, etc. */
+    /* Header comments for labeling functions, classes, test samples, etc */
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
     /**
-     * multi-line
-     * marginal notes
+     * Multiline
+     * comments
      */
     ```
 
 === "Zig"
 
     ```zig title=""
-    // Header comments for labeling functions, classes, test samples, etc.
+    // Header comments for labeling functions, classes, test samples, etc
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
-    // Multi-line
-    // Annotation
+    // Multiline
+    // comments
     ```
 
-## Learn Efficiently In Animated Graphic Solutions
+## Efficient Learning via Animated and Graphical Illustrations
 
-Compared with text, videos and pictures have a higher degree of information density and structure and are easier to understand. In this book, **key and difficult knowledge will be presented mainly in the form of animations and graphs**, while the text serves as an explanation and supplement to the animations and graphs.
+Compared with text, videos and pictures have a higher density of information and are more structured, making them easier to understand. In this book, **essential and complex concepts are primarily presented through animations and illustrations**, while the text serves as an explanation and supplement to the animations and graphs.
```

Comment:
Compared to text, videos and images have higher information density and are more structured, thus easier to understand. In this book, **key and difficult concepts are mainly presented through animations and illustrations**, with text serving as explanations and supplements.